horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/TestServiceImpl.java

```diff
@@ -157,9 +154,10 @@ public Test getByNameOrId(String input){
       TestDAO test = null;
       if (input.matches("-?\\d+")) {
          int id = Integer.parseInt(input);
+         // there could be some issue if name is numeric and corresponds to another test id
```

Comment:
Can you open a new issue for this?

Reply:
Created --> https://github.com/Hyperfoil/Horreum/issues/1670